### ISSUE TITLE
encyc-core always updating 197 articles

### ISSUE THREAD

**Issue by gjost** [denshoproject/encyc-core#39]
197 articles are always updated every time `cron` invokes `encyc articles`, as can be seen by looking at `/var/log/encyc/core-syncwiki.log`.

```
encyc.cli.articles()
231: encyc.publish.articles()
942: encyc.models.elastic.Elasticsearch.articles_to_update()
921: encyc.models.elastic.Elasticsearch._new_update_deleted()
```
This code is used to generate the list, across the various modules:
```
# encyc.cli
from encyc.models.elastic import Author, Page, Source
from encyc.models.legacy import Page as LegacyPage, Proxy
from encyc import wiki
# encyc.cli.articles()
mw = wiki.MediaWiki()
mw_author_titles = Proxy.authors(mw, cached_ok=False)
mw_articles = Proxy.articles_lastmod(mw)
es_articles = Page.pages()
# encyc.publish.articles()
es_objects = es_articles.objects
es_articles = es_objects
# encyc.models.elastic.py
from datetime import datetime
from dateutil import parser
#  articles_to_update()
mw_pages = {a['title']: a for a in mw_articles if a['title'] not in mw_author_titles}
es_objects = {a.title: a for a in es_articles}
# _new_update_deleted()
def force_dt(val):
    if isinstance(val, datetime):
        return val
    return parser.parse(val)
new = [
    title for title in mw_pages.keys() if not title in es_objects.keys()
]
updated = [
    es.title for es in es_objects.values()
    if mw_pages.get(es.title)
    and (force_dt(mw_pages[es.title]['lastmod']) > force_dt(es.modified))
]
```

**Comment by gjost** [comment]
The problem is not the code that is gathering the list.
The problem is that somehow these articles are not getting updated in Elasticsearch.

**Comment by gjost** [comment]
Fixed in commit `d6326a0` and merged in to `develop` branch.

**Comment by gjost** [comment]
Merged in to `master` as of commit `73b6030`.
